modules/openapi-generator/src/main/resources/typescript/api/api.mustache

```diff
@@ -1,6 +1,9 @@
 // TODO: better import syntax?
 import {BaseAPIRequestFactory, RequiredError, COLLECTION_FORMATS} from './baseapi{{importFileExtension}}';
-import {Configuration} from '../configuration{{importFileExtension}}';
+import {Configuration,ConfigurationOptions} from '../configuration{{importFileExtension}}';
```

Comment:
is this import still needed? the ConfigurationOptions are not used in the code, are they?

Reply:
removed!